ptvsd/wrapper.py

```diff
@@ -790,6 +791,30 @@ def process_launch_arguments(self):
             redirect_output = ''
         self.pydevd_request(pydevd_comm.CMD_REDIRECT_OUTPUT, redirect_output)
 
+    def __parse_launch_option_value(self, value):
+        if value == 'True' or value == 'False':
+            return bool(value)
+        return unquote(value)
+
+    def __parse_launch_options(self, launch_options):
+        """VS options semicolon separated key=value pairs
```

Comment:
I'd remove this message. There's no reason VSC or another debugger can't use this format. If it did then one would assume this was being used only by VS.